index.html

```diff
@@ -0,0 +1,185 @@
+        <div class="col-md-6 col-lg-4 programs-container">
+          <div class="programs d-flex flex-md-column mt-3">
+            <i class="fa-solid fa-person-chalkboard"></i>
+            <h6>Keynote I/O</h6>
+            <p>
+              Tune in to find out how
+              we're furthering our mission
+            </p>
+          </div>
+        </div>
+        <div class="col-md-6 col-lg-4 programs-container">
+          <div class="programs d-flex flex-md-column mt-3">
+            <i class="fa-brands fa-android"></i>
+            <h6>Android development</h6>
+            <p>
+              Hear the latest news on everything happening in the
+            </p>
+          </div>
+        </div>
+        <div class="col-md-6 col-lg-4 programs-container">
+          <div class="programs d-flex flex-md-column mt-3">
+            <i class="fa-solid fa-cloud"></i>
+            <h6>Google Cloud</h6>
+            <p>
+              Learn how Google Cloud and Workspace teams are
+            </p>
+          </div>
+        </div>
+        <div class="col-md-6 col-lg-4 programs-container">
+          <div class="programs d-flex flex-md-column mt-3">
+            <i class="fa-solid fa-fire"></i>
+            <h6>Google Firebase</h6>
+            <p>
+              Learn how to build and run apps users love with Firebase
+            </p>
+          </div>
+        </div>
+        <div class="col-md-6 col-lg-4 programs-container">
+          <div class="programs d-flex flex-md-column mt-3">
+            <i class="fa-solid fa-brain"></i>
+            <h6>AI</h6>
+            <p>
+              Learn what Google is doing in the AI and machine learning space
+            </p>
+          </div>
+        </div>
+        <div class="col-md-6 col-lg-4 programs-container">
+          <div class="programs d-flex flex-md-column mt-3">
+            <i class="fa-solid fa-vr-cardboard"></i>
+            <h6>AR</h6>
+            <p>
+              Get the latest on our AR developer tools, including the new ARCore
+            </p>
+          </div>
+        </div>
+      </div>
+      <button type="button" class="btn btn-outline btn-lg">Join Google I/O conference 2022</button>
+    </div>
+  </section>
+  <!-- Featured speakers section -->
+  <section id="speakers" class="container-fluid">
+    
+  </section>
+ 
+  <!--  partner section -->
+   <div class="container-fluid partners-container">
+    <h1 class="text-center">Partners</h1>
+    <p class="text-center dash">____<p>
+    <div class="container d-flex justify-content-around partners-1">
+      <i class="fa-brands fa-aws" id="aws"></i>
+      <i class="fa-brands fa-youtube" id="youtube"></i>
+      <i class="fa-brands fa-facebook" id="facebook"></i>
+    </div>
+    <div class="container d-flex justify-content-around partners-2">
+      <i class="fa-brands fa-twitter" id="twitter"></i>
+      <i class="fa-brands fa-firefox" id="firefox"></i>
+    </div>
+  </div> 
+  <!-- footer section --> 
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you ensure that you remove the horizontal navigation as this is caused when an element length is overflowing horizontally on the page. 
- You can try this CSS snippet to inspect your code and see which element is overflowing horizontally:
`* { border: 1px solid black; }`

https://user-images.githubusercontent.com/52934128/169326031-1fd67078-600f-47a2-adf2-1f295cd2f729.mp4